contributing/lts-release.md

```diff
@@ -0,0 +1,29 @@
+# Long-Term Stable (LTS) Release Channel
+
+The **lts** release channel provides a previous **stable** build for four-week intervals. Every four weeks, the current **stable** release is promoted to **lts**. This is not recommended for all AMP publishers, but is provided so publishers performing a QA cycle may do so less often. Individual pages can explicitly opt into **lts**.
+
+> Note: The LTS release channel is available only to AMP HTML pages; it is not available to AMP Actions, AMP for Email, AMP Stories, or AMP for Ads.
```

Comment:
I'd specify "AMP HTML pages" as "AMP Websites" as, technically, AMP Stories are also AMP HTML pages. Or as "AMP HTML pages used in websites"